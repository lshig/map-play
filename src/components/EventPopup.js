import PropType from 'prop-types';
import React from 'react';

export default function EventPopup({ text }) {
  return <div className="popup">{text}</div>
};

EventPopup.propTypes = {
  text: PropType.string.isRequired
};
