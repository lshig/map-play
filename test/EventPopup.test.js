import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import EventPopup from '../src/components/EventPopup';

describe('EventPopup', () => {
  describe('basic', () => {
    const wrapper = shallow(
      <EventPopup text="Okinawa milk tea with egg pudding. 80% sweetness. 30% ice." />
    );

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
