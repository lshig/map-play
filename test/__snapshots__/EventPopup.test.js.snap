// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventPopup basic renders 1`] = `
<div
  className="popup"
>
  Okinawa milk tea with egg pudding. 80% sweetness. 30% ice.
</div>
`;
